docs/tutorials/articles/understanding_gui/index.md

```diff
@@ -37,13 +37,13 @@ accordingly.
 
 ### Installation
 
-Ensure you have Python 3.8 or newer, then install Taipy:
+Ensure you have Python 3.9 or newer, then install Taipy:
 
 ```bash
 pip install taipy
 ```
 
-For the NLP feature (Python 3.8 to 3.10), install:
+For the NLP feature (Python 3.9 to 3.10), install:
```

Comment:
Is this still true? With Python 3.12 out?

Reply:
It's for PyTorch. I'm not sure if it remains true till this day. Let me check